Generate the `activation` ping and send it with Glean (#1707)
mozilla-mobile/fenix: docs/telemetry.md

Fenix sends event pings that allows us to measure feature performance. These are defined inside the [`metrics.yaml`](https://github.com/mozilla-mobile/fenix/blob/master/app/metrics.yaml) file.

## Activation

> Fenix sends an activation ping once, at startup. The ping is documented [`here`](activation.md) file.

Reviewer: ```suggestion
Fenix sends an activation ping once, at startup. Further documentation can be found in [Activation Ping](activation.md).
```
